package/MDAnalysis/make_bundle.py

```diff
@@ -52,39 +54,45 @@ def make_bundle(*args, **kwargs):
     # TODO - use Group
     # TODO - careful about overwriting existing sims/category values
 
-    names = kwargs.get('names', map(str, range(len(args[0]))))
-    if len(args) == 2:
+    if not names:
+        # if names not provided, use the index for name
+        names = map(str, range(len(args[0])))
+    if not isinstance(simulations, list): 
```

Comment:
No need to check here. We only want something that implements the random access api. A tuple or numpy array should also work fine.